### ISSUE TITLE
MGHPCC 2023 Shutdown Bring systems back up (June 7th)

### ISSUE THREAD

**Issue by joachimweyl** [CCI-MOC/ops-issues#935]
Gently bring up
- [ ]  ovirts VMs
- [ ]  production ceph cluster.
- [ ]  research ceph
- [ ]  maas nodes
- [ ]  all elastic nodes
- [ ]  kumo nodes
- [ ]  ocp-staging
- [ ]  ocp-prod
- [ ]  openstack
- [ ]  moc-infra
- [ ]  smaug

**Comment by naved001** [comment]
Things I won't bring back up:

* Smaug cluster (OCT5 nodes and 3 controllers in OCT10 (4,5,6)
* Openstack Compute nodes

**Comment by naved001** [comment]
Some notes I took while bringing the systems up:

Power on kzn-ipmi-gw from ovirt.

Power on freeipa VM so I can login with my ipa credentials

Power on HA NFS servers

Power on VMs:
* foreman
* various DNS VMs
* dhcp
* ovpn and router
* sso -> broken


Power on the ceph cluster:
* monitors
* osd servers
* once all OSD servers are up, unset the ceph flags
```
ceph osd set noout
ceph osd set norecover
ceph osd set norebalance
ceph osd set nobackfill
ceph osd set nodown
ceph osd set pause
```

* troubleshoot ceph
- one server failed to boot
- one monitor too
    * L2/L3 cache error was detected on the raid controller
- foreman didn't give out IP on 172.16.0.0/19 network to these hosts

Power on the remaining VMs on ovirt that rely on ceph cluster:
* maas
* zabbix
* etc

Power on research ceph
* monitors first, from ovirt
* OSD hosts
    - 15.22 - ipmi pass got reset, so it failed to power on
* 5 OSDs are down :-<

Power on moc-infra
* powered on the nodes

Power on ocp-prod
* powreed on the nodes

Power on ocp-staging
* powered on the nodes
* one controller, a kumo blade, is unavailable.

Powered on openstack controllers only

**Comment by hakasapl** [comment]
CloudLab is back up, as is the CASA cluster and the OCT network.

CloudLab had some network loop issues which I resolved.

**Comment by joachimweyl** [comment]
This appears to be done. Are there any last steps?

**Comment by hakasapl** [comment]
No, this should be complete, the last 2 are indicated to be taken care of by others.
